src/live.rs

```diff
@@ -26,7 +26,22 @@ use crate::io::*;
 
 const FILENAME: &str = "config.ign";
 
-pub fn iso_embed(config: &IsoEmbedConfig) -> Result<()> {
+pub fn iso_embed(config: &IsoIgnitionEmbedConfig) -> Result<()> {
+    eprintln!("iso embed is deprecated; use iso ignition embed.  Continuing.");
```

Comment:
Minor/optional:

```suggestion
    eprintln!("`iso embed` is deprecated; use `iso ignition embed`.  Continuing.");
```

?

Aside: I'm kinda surprised `clap` doesn't have a built-in deprecation mechanism to emit this warning.